app_flutter/lib/service/mock_cocoon.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:fixnum/fixnum.dart';
+
+import 'package:cocoon_service/protos.dart'
+    show Commit, CommitStatus, Stage, Task;
+
+import 'cocoon.dart';
+
+class MockCocoonService implements CocoonService {
```

Comment:
This class doesn't use Mockito, so it is a Fake and not a Mock - it should be named appropriately.

Reply:
Thank you! I just got a good idea on the difference between the two :)

Reply:
See more at https://martinfowler.com/bliki/TestDouble.html